src/components/project_search/ProjectField.tsx

```diff
@@ -0,0 +1,112 @@
+import ProjectOption from './ProjectOption';
+import { DescriptionFieldNames } from '../../types/publicationFieldNames';
+import { BackendTypeNames } from '../../types/publication_types/commonPublication.types';
+
+const StyledSearchIcon = styled(SearchIcon)`
+  margin-left: 0.5rem;
+  color: ${({ theme }) => theme.palette.text.disabled};
+`;
+
+export const ProjectSearch: FC = () => {
+  const { t } = useTranslation('publication');
+  const [projects, isLoadingProjects, handleNewSearchTerm] = useFetchProjects('');
+
+  const debouncedSearch = useCallback(
+    debounce((searchTerm: string) => {
+      handleNewSearchTerm(searchTerm);
+    }),
+    []
+  );
+
+  return (
+    <Field name={DescriptionFieldNames.PROJECT}>
+      {({ field, form: { setFieldValue } }: FieldProps<ResearchProject>) => (
+        <Autocomplete
+          options={projects}
+          noOptionsText={t('common:no_hits')}
+          loadingText={`${t('common:loading')}...`}
+          getOptionLabel={(option) => getProjectTitle(option)}
+          onInputChange={(_, newInputValue) => {
+            debouncedSearch(newInputValue);
+          }}
+          onChange={(_, value) => {
+            const projectToPersist = value[0] ? convertToProject(value[0]) : undefined;
+            setFieldValue(field.name, projectToPersist);
+          }}
+          multiple
+          defaultValue={field.value ? [field.value].map((project) => convertToCristinProject(project)) : []}
+          renderTags={(value, getTagProps) =>
+            value.map((option, index) => <ProjectChip project={option} {...getTagProps({ index })} />)
+          }
+          getOptionDisabled={(option) => field.value?.id === option.cristinProjectId}
+          loading={isLoadingProjects}
+          renderOption={(option, state) => <ProjectOption project={option} state={state} />}
+          renderInput={(params) => (
+            <TextField
+              {...params}
+              variant="outlined"
+              fullWidth
+              placeholder={t('description.search_for_project')}
+              onKeyDown={(event: KeyboardEvent) => {
+                if (event.key === 'Backspace') {
+                  // Disable removing chips with backspace
+                  event.stopPropagation();
+                }
+              }}
+              InputProps={{
+                ...params.InputProps,
+                startAdornment: (
+                  <>
+                    {params.InputProps.startAdornment}
+                    <StyledSearchIcon />
+                  </>
+                ),
+                endAdornment: (
+                  <>
+                    {isLoadingProjects && <CircularProgress color="inherit" size={20} />}
+                    {params.InputProps.endAdornment}
+                  </>
+                ),
+              }}
+            />
+          )}
+        />
+      )}
+    </Field>
+  );
+};
+
+export default ProjectSearch;
+
```

Comment:
vi må ha noe mapping som under her frem til vi har en felles modell for prosjekt: https://unit.atlassian.net/browse/NP-1718

Reply:
det blir også noen endringer (primært forenklinger) når datamodellen får støtte for flere prosjekter